cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -22,7 +22,7 @@ Contributors:
 -   Robert Oschwald [CAS 3.0]
 -   Misagh Moayyed
 
-Version: 3.0.1
+Version: 3.0.2
```

Comment:
Well, since the parameter is optional per semantic versioning, I *think* this can be applied as a "patch release" to the protocol. A minor change would be more suitable if we had more drastic changes to the protocol.